manuscript/Manuscript.qmd

```diff
@@ -2085,9 +2085,9 @@ Relevant ecosystem condition variables are recorded for each delineated ecosyste
 ## Indicator values in the three pilot municipalities
 
 The indicator values for our three indicators across the three municipalities showed in general a good condition in the mire ecosystems (@fig-EEA-ridgePlot).
-The exception was for variable *Trenching* in Nordre Follo, where the indicator value was 0.25; well below the threshold for good ecosystem condition (0.6).
+The exception was for variable *Trenching* in Nordre Follo, where the indicator value was 0.25; well below the threshold for good ecosystem condition (0.6) (REF Jakobsson 2020, 2021).
```

Comment:
I think to base the statement that 0.6 equals the threshold between good and degregated ecosystems, we should instead cite SEEA EA